plugins/cache_promote/policy.h

```diff
@@ -28,6 +28,10 @@ extern int TXN_ARG_IDX;
 #define MAX_STAT_LENGTH (1 << 8)
 extern const char *PLUGIN_NAME;
 
+extern DbgCtl cache_promote_dbg_ctl;
+
+#define DBG(...) Dbg(cache_promote_dbg_ctl, __VA_ARGS__)
```

Comment:
Why is this plugin converted with a macro rather than `Dbg`?

Reply:
I think I did this before I thought of the other approach with `using namespace`.